src/render/vulkan/SDL_render_vulkan.c

```diff
@@ -1642,13 +1746,21 @@ static VkResult VULKAN_CreateDeviceResources(SDL_Renderer *renderer, SDL_Propert
         VkDeviceCreateInfo deviceCreateInfo = { 0 };
         static const char *const deviceExtensionNames[] = {
             VK_KHR_SWAPCHAIN_EXTENSION_NAME,
+             /* VK_KHR_sampler_ycbcr_conversion + dependent extensions */
+            VK_KHR_SAMPLER_YCBCR_CONVERSION_EXTENSION_NAME,
+            VK_KHR_MAINTENANCE1_EXTENSION_NAME,
+            VK_KHR_BIND_MEMORY_2_EXTENSION_NAME,
+            VK_KHR_GET_MEMORY_REQUIREMENTS_2_EXTENSION_NAME,
         };
+        /* Check if YcBcr extensions are supported */
+        rendererData->supportsKHRSamplerYcBcrConversion = rendererData->supportsKHRGetPhysicalDeviceProperties2 &&
 
+        VULKAN_DeviceExtensionsFound(rendererData, 4, &deviceExtensionNames[1]);
```

Comment:
Can you change this to if (...)? It's not immediately clear what's happening, especially given the indentation.